frame/contracts/src/wasm/runtime.rs

```diff
@@ -2444,4 +2456,36 @@ pub mod env {
 			Err(_) => Ok(ReturnCode::EcdsaRecoverFailed),
 		}
 	}
+
+	/// Returns the number of times the currently executing contract exists on the call stack in
+	/// addition to the calling instance.
+	///
+	/// # Return Value
+	///
+	/// Returns 0 when there is no reentrancy.
+	#[unstable]
+	#[prefixed_alias]
```

Comment:
we don't need the prefixed alias for the new functions; it's only needed for old ones for backwards compatibility
```suggestion
```